packages/documentation/copy/en/get-started/TS for the New Programmer.md

```diff
@@ -29,7 +29,7 @@ There are many developers these days that use _only_ JavaScript to program their
 To summarize, we have a language that was designed for quick uses, and then grew to a full-fledged tool to write applications with millions of lines.
 Every language has its own _quirks_ — oddities and surprises, and JavaScript's humble beginning makes it have _many_ of these. Some examples:
 
-- JavaScript's equality operator (`==`) _coerces_ its arguments, leading to unexpected behavior:
+- JavaScript's equality operator (`==`) attempts to _coerce_ the values, leading to unexpected behavior:
```

Comment:
Given that this is for new programmers, I think we're expecting them to know what coercing types is in both versions we could go simpler:

```suggestion
- JavaScript's equality operator (`==`) acts quite loose with types, leading to unexpected behavior:
```

Reply:
@orta good!